src/assets/scripts/client/constants/themes/default/modules/dataBlock.js

```diff
@@ -0,0 +1,168 @@
+import { COLOR } from '../colors';
+
+// TODO: Instead of individual colors, can we somehow specify an alpha value
```

Comment:
thats a possibility. or you could define base `rgb` colors and create a function that returns an `rgba` color with a passed opacity value